### DISCUSSION TITLE
Docker version of BiblioteQ.

### DISCUSSION THREAD

**Discussion by swiss-knight** [textbrowser/biblioteq#92] [Ideas]
Would it be possible to fully dockerize biblioteq, and provide in addition a `docker-compose.yml` file to startup the app using a simple as possible example of needed environment variables (e.g. `.env.sample`) along with clear explanation for each of these in a README?

**Comment by textbrowser** [comment]
Sorry about the delay. Probably. What's the benefit of Docking BQ?

**Reply by g0rge0us** [reply]
> Sorry about the delay. Probably. What's the benefit of Docking BQ?

Well, it will be much easier to deploy the server app/db and (I guess) to maintain everything from a browser and mobile app.

**Reply by textbrowser** [reply]
I see. Another bundle. :P I'm not so interested in writing more bundles. If someone will do it and promise to support it, sure!
